marianas_collapsing_first_pass_1.8.1/marianas_first_pass.cwl

```diff
@@ -0,0 +1,158 @@
+  - id: min_mapping_quality
+    type: int
+    inputBinding:
+      position: 3
+  - id: min_base_quality
+    type: int
+    inputBinding:
+      position: 4
+  - id: mismatches
+    type: int
+    inputBinding:
+      position: 5
+  - id: wobble
+    type: int
+    inputBinding:
+      position: 6
+  - id: min_consensus_percent
+    type: int
+    inputBinding:
+      position: 7
+  - id: reference_fasta
+    type: File
+    inputBinding:
+      position: 8
+    secondaryFiles:
+      - .fai
+      - ^.dict
+outputs:
+  - id: first_pass_output_file
+    type: File
+    outputBinding:
+      glob: first-pass.txt
+  - id: alt_allele_file
+    type: File
+    outputBinding:
+      glob: first-pass-alt-alleles.txt
+  - id: first_pass_insertions
+    type: File
+    outputBinding:
+      glob: first-pass-insertions.txt
+  - id: first_pass_output_dir
+    type: Directory
+    outputBinding:
+      glob: .
+arguments:
+  - position: 0
+    valueFrom: '-server'
+  - position: 0
+    valueFrom: |-
+      ${
+        if(inputs.memory_per_job && inputs.memory_overhead) {
+          if(inputs.memory_per_job % 1000 == 0) {
+            return "-Xms" + (inputs.memory_per_job/1000).toString() + "G"
+          }
+          else {
+            return "-Xms" + Math.floor((inputs.memory_per_job/1000)).toString() + "G"
+          }
+        }
+        else if (inputs.memory_per_job && !inputs.memory_overhead){
+          if(inputs.memory_per_job % 1000 == 0) {
+            return "-Xms" + (inputs.memory_per_job/1000).toString() + "G"
+          }
+          else {
+            return "-Xms" + Math.floor((inputs.memory_per_job/1000)).toString() + "G"
+          }
+        }
+        else if(!inputs.memory_per_job && inputs.memory_overhead){
+          return "-Xms8G"
+        }
+        else {
+            return "-Xms8G"
+        }
+      }
+  - position: 0
+    valueFrom: |-
+      ${
+        if(inputs.memory_per_job && inputs.memory_overhead) {
+          if(inputs.memory_per_job % 1000 == 0) {
+            return "-Xmx" + (inputs.memory_per_job/1000).toString() + "G"
+          }
+          else {
+            return "-Xmx" + Math.floor((inputs.memory_per_job/1000)).toString() + "G"
+          }
+        }
+        else if (inputs.memory_per_job && !inputs.memory_overhead){
+          if(inputs.memory_per_job % 1000 == 0) {
+            return "-Xmx" + (inputs.memory_per_job/1000).toString() + "G"
+          }
+          else {
+            return "-Xmx" + Math.floor((inputs.memory_per_job/1000)).toString() + "G"
+          }
+        }
+        else if(!inputs.memory_per_job && inputs.memory_overhead){
+          return "-Xmx8G"
+        }
+        else {
+            return "-Xmx8G"
+        }
+      }
+  - position: 0
+    prefix: '-cp'
+    valueFrom: /usr/local/bin/Marianas-1.8.1.jar
+  - position: 0
+    valueFrom: org.mskcc.marianas.umi.duplex.DuplexUMIBamToCollapsedFastqFirstPass
+requirements:
+  - class: ResourceRequirement
+    ramMin: |-
```

Comment:
As we now know that ramMin does not work with Javascript in Toil 3.19 can you comment this out and rewrite `ramMin: 20000`